docs/synctl-create-test.md

```diff
@@ -44,19 +44,22 @@ synctl create test [options]
 ```
 ### Options for API Script test
 ```  
-    -f, --from-file <file>              Synthetic script/payload, support json, js and side file, e.g, script.js
+    -f, --from-file <file>              load synthetic test payload from file (.json)
```

Comment:
since "-f, --from-file" is available for all test types, let us move it to "Common options", and no need to replicate in all test types